hapi/hapi/build.gradle.kts

```diff
@@ -3,8 +3,8 @@ plugins {
     id("org.hiero.gradle.module.library")
     id("org.hiero.gradle.feature.protobuf")
     id("org.hiero.gradle.feature.test-fixtures")
-    id("com.hedera.pbj.pbj-compiler") version "0.11.15"
-    // ATTENTION: keep in sync with pbj version in 'hiero-dependency-versions/build.gradle.kts'
+    // ATTENTION: keep pbj version in sync with 'hiero-dependency-versions/build.gradle.kts'
+    id("com.hedera.pbj.pbj-compiler") version "0.12.0"
```

Comment:
FYI: I am thinking about a good way to get rid of defining the version here. If the _otter test_ project also starts to use PBJ, the version should be maintained centrally.